README.md

```diff
@@ -384,20 +384,57 @@ public CloudTasksClient getService() throws IOException {
 [`OperationFuture`](https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.longrunning.OperationFuture).
-When you call `get()` on the `OperationFuture` we poll the operation endpoint to
-check on the operation. These polling operations have a default timeout that
-varies from service to service and will throw a `java.util.concurrent.CancellationException`
-with the message: `Task was cancelled.` after that timeout has been reached.
+When calling `get()` on the `OperationFuture`, the client library will poll the operation to
+check the operation's status.
+
+### LRO Timeouts
+The polling operations have a default timeout that varies from service to service.
+The library will throw a `java.util.concurrent.CancellationException` with the message:
+`Task was cancelled.` if the timeout exceeds the operation. A `CancellationException`
+does not mean that the backend GCP Operation was cancelled. This exception is thrown from the
+client library when it has exceeded the total timeout without receiving a successful status from the operation.
+Our client libraries respect the configured values set in the OperationTimedPollAlgorithm for each RPC.
+
+Note: The client library handles the Operation's polling mechanism for you. By default, there is no need
+to manually poll the status yourself.
+
+### Default LRO Values
+Each LRO RPC has a pre-configured default values. You can find these values by 
+searching in each Client's `StubSettings`'s class. The default LRO settings are initialized
+inside the `initDefaults()` method in the nested Builder class.
+
+For example, in AI-Platform v3.24.0, the default [OperationTimedPollAlgorithm](https://github.com/googleapis/google-cloud-java/blob/9ae786d1acdc7354adf86b78691570668caa293d/java-aiplatform/google-cloud-aiplatform/src/main/java/com/google/cloud/aiplatform/v1/stub/EndpointServiceStubSettings.java#L755-L765) 
+has these default values:
+```java
+OperationTimedPollAlgorithm.create(
+    RetrySettings.newBuilder()
+        .setInitialRetryDelay(Duration.ofMillis(5000L))
+        .setRetryDelayMultiplier(1.5)
+        .setMaxRetryDelay(Duration.ofMillis(45000L))
+        .setInitialRpcTimeout(Duration.ZERO)
+        .setRpcTimeoutMultiplier(1.0)
+        .setMaxRpcTimeout(Duration.ZERO)
+        .setTotalTimeout(Duration.ofMillis(300000L))
+        .build())
+```
+Both retries and LROs share the same RetrySettings class. Note the corresponding link:
+- Total Timeout (Max Time allowed for polling): 5 minutes
+- Initial Retry Delay (Initial delay before first poll): 5 seconds
+- Max Retry Delay (Maximum delay between each poll): 45 seconds
+- Retry Delay Multiplier (Multiplier value to increase the poll delay): 1.5
 
-### Configuring LRO Timeouts
+The RPC Timeout values have no use in LROs and can be omitted or set to the default values
+(`Duration.ZERO` for Timeouts or `1.0` for the multiplier)
 
+### Configuring LRO Timeouts
 ```java
```

Comment:
How about adding a sentence?

"To configure the LRO timeouts, set the values in `RetrySettings`. For exapmle:"

Reply:
will do!